### PR TITLE
Dockerfile fix

### PR DESCRIPTION
**Pull request by Knuspel**
I updated the Dockerfile to the proper URLs so that it uses the current Versions of  the Manager and factorio itself.

### CODE REVIEW

**Comment by knoxfighter** [comment]
Good catch, we also hadn't it on the develop branch. I changed your base, please fix the merge conflicts.
In the case you didn't know, in the develop branch, also docker-compose files with the new UI are available. The current develop branch is also pushed to dockerhub with the tag `ofsm/ofsm:develop`.

**Comment by Knuspel** [comment]
Fixed the conflicts.
Should work now.

File: docker/Dockerfile [hunk 1]
```diff
@@ -17,12 +17,17 @@ RUN apk add --no-cache curl tar xz unzip jq
...
          --output /tmp/factorio-server-manager-linux_${MANAGER_VERSION}.zip \
     && unzip /tmp/factorio-server-manager-linux_${MANAGER_VERSION}.zip \
     && rm /tmp/factorio-server-manager-linux_${MANAGER_VERSION}.zip \
     && mv factorio-server-manager fsm
 
+# Setup Nginx
+    mkdir -p /run/nginx && \
+    chown nginx:root /var/lib/nginx
+
+
```
Comment (knoxfighter) [inline docker/Dockerfile:30]: remove nginx, no nginx is installed in the container

Comment (Knuspel) [reply docker/Dockerfile]: done.

**Pull request merged**